scipy/stats/tests/test_bootstrap.py

```diff
@@ -54,6 +58,68 @@ def statistic(x, y, axis):
     with pytest.raises(ValueError, match=message):
         bootstrap(([1, 2, 3],), np.mean, random_state='herring')
 
+@pytest.mark.parametrize("method", ['basic', 'percentile', 'BCa'])
```

Comment:
Two blank lines before top-level `def`.
```suggestion

@pytest.mark.parametrize("method", ['basic', 'percentile', 'BCa'])
```

Reply:
Will do.